### PR TITLE
Document IAM policies and link to sample manifest

### LINKED ISSUE #19: Document mandatory IAM roles
Users not having the required IAM roles set up in their clusters are likely to face issues like https://github.com/weaveworks/eksctl/issues/1237. We ought to at least document these.

### PR DESCRIPTION
**Pull request by marccarre**
Preview available here: https://github.com/weaveworks/eks-quickstart-app-dev/blob/issues/19-doc-iam-policies/README.md
Depends on https://github.com/weaveworks/eksctl/pull/1249
Fixes #19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,16 @@ configured by [eksctl](https://eksctl.io) through GitOps.
...
 
+## Pre-requisites
+
+A running EKS cluster with [IAM policies](https://eksctl.io/usage/iam-policies/) for:
+
+- ALB ingress
+- auto-scaler
+- CloudWatch
+
+[Here](https://github.com/weaveworks/eksctl/blob/master/examples/eks-quickstart-app-dev.yaml) is a sample `ClusterConfig` manifest for such policies enabled.
```
Comment (errordeveloper) [inline README.md]: _"... that shows how to enable all of these policies."_

We might want to add a note to that these will be scoped to a particular nodegroup, so a cluster with multiple nodegroups will require all of the add-on pods in question to have node selector at very least.

Comment (marccarre) [reply README.md]: > a cluster with multiple nodegroups will require all of the add-on pods in question to have node selector at very least.

@errordeveloper why is this?
What is the typical use-case for a cluster with multiple nodegroups?
Shouldn't we add such labels in our templates, add an optional flag for the target nodegroup's name, and deal with this when applying the profile in `eksctl`?
(If so, feel free to answer the above questions directly in a new issue. 🙇)

Comment (marccarre) [reply README.md]: Note that I've added some notes (PTAL 🙇) about the nodegroup scope and what to do with multiple nodegroups, but it is only based on my assumptions and my (limited) understanding. (Hence the aforementioned questions.)

**Comment by marccarre** [comment]
> ilya:
> ok, actually, I think #22 looks good

**Pull request merged**